perceval/serialization/deserialize.py

```diff
@@ -41,8 +41,11 @@ def deserialize_matrix(pb_mat: Union[str, pb.Matrix]) -> Matrix:
     if not isinstance(pb_mat, pb.Matrix):
         pb_binary_repr = pb_mat
         pb_mat = pb.Matrix()
-        assert pb_binary_repr.startswith(":PCVL:Matrix:")
-        pb_mat.ParseFromString(b64decode(pb_binary_repr[13:]))
+        if isinstance(pb_binary_repr, bytes):
+            pb_mat.ParseFromString(pb_binary_repr)
+        else:
+            assert pb_binary_repr.startswith(":PCVL:Matrix:")
```

Comment:
Please put the string in a dedicated variable and use the variable size in the next line.